cookbook/security/entity_provider.rst

```diff
@@ -583,6 +583,11 @@ The code below shows the implementation of the
         }
     }
 
+.. tip::
+
+    Don't forget to add the repository class to the 
```

Comment:
perhaps here there is no cause or reasoning as to why we shouldn't forget. Maybe explain don't forget else it will not be hooked or it will just use the default class or other worst.